endoreg_db/utils/__init__.py

```
@@ -4,20 +4,30 @@
+    DJANGO_NAME_SALT
 )
 from .names import (
     create_mock_patient_name,
     create_mock_examiner_name,
     guess_name_gender,
 )
 
+from .paths import data_paths
+
+from .env import (
+    get_env_var,
```

Ruff (0.8.2)

`.env.get_env_var` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)